src/components/common/ProductBox/ProductBox.js

```diff
@@ -47,6 +47,11 @@ const ProductBox = ({ name, price, promo, stars }) => (
         </Button>
       </div>
       <div className={styles.price}>
+        {priceOld > 0 && (
```

Comment:
A dlaczego priceOld miała by być 0? Jeśli nie było starej ceny to jej po prostu nie bedzie czyli `undefined`

Reply:
Faktycznie, niepotrzebne linijki, poprawione